Cyberwatch/cyberwatch-detection/_meta/fields.yml

```
@@ -0,0 +1,69 @@
+cyberwatch.active:
```

```suggestion
cyberwatch.vas.active:
```
Based on `cyberwatch.vas.fixed_at`. What do you think @squioc ?

---

Yes. I agree with you. I think all custom fields should be prefixed with `cyberwatch.vas`.